src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -169,7 +174,34 @@ public HttpRouteCollection Routes
                 };
                 using (logger.BeginScope(scopeState))
                 {
+                    JObject coldStartData = null;
+                    if (request.IsColdStart())
+                    {
+                        var dispatchStopwatch = request.GetPropertyOrDefault<Stopwatch>(ScriptConstants.AzureFunctionsColdStartKey);
```

Comment:
Nit: GetPropertyOrDefault will give you a null if it doesn't exist, right? I know this is paranoid but ideally you'd do a null check here before calling Stop().

Reply:
fixed